node_frequency: 50.0
